zb-classifier.js

```diff
@@ -1240,6 +1275,14 @@ class ZigbeeClassifier {
     this.addZoneTypeProperty(node, propertyName, propertyDescr);
   }
 
+  initDoorLock(node, doorLockEndpoint) {
+    // TODO: Replace with DoorLock type
+    node.type = Constants.THING_TYPE_ON_OFF_SWITCH;
```

Comment:
We don't really need to be setting a `type` anymore, as it's no longer in the spec.